cmake/FindPVRTexLib.cmake

```diff
@@ -22,7 +22,8 @@
 #  PVRTEXLIB_LIBRARIES - The libraries needed to use PVRTexLib
 
 IF (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-  SET( PVRTEXLIB_ROOT "/Applications/Imagination/PowerVR_Graphics/PowerVR_Tools/PVRTexTool/Library" )
+  SET( PVRTEXLIB_ROOT "/Applications/Imagination/PowerVR_Graphics/PowerVR_Tools/PVRTexTool/Library"
+       CACHE PATH "Location of the PVRTexTool shared library" )
```

Comment:
This documentation string is a bit misleading, since the shared library is actually under a subdirectory. (e.g. Linux_x86_64, Windows_x86_64) It would probably be better to say this is the path to the PVRTexTool/Library directory.

Reply:
Fixed. I went with "Location of the PVRTexTool library platform subdirectories", which I defined in excruciating detail in the readme. I think "path to the PVRTexTool/Library directory" is not great because it implies that the give path has to contain a folder PVRTexTool/Library.